### ISSUE TITLE
Wish: Provide a project or package flag to prevent format_spec_file execution

### ISSUE THREAD

**Issue by krop** [openSUSE/obs-service-format_spec_file#71]
tl;dr; obs_service_format_spec_file does weird cosmetic and unwanted changes automatically and there's no way to skip its usage at package or project level.

Among issues caused by this tool:
- Reordering license tags and sometimes changing their meaning
- Randomly adds or remove empty lines if a package description is followed by a commend

spec-cleaner-format_spec_file is probably not behaving better, since it's also making unwanted changes, such as moving conditional blocks below other BuildRequires/Requires/recommends... lines.

While convenient, we can't ask occasional contributors to use `osc ci --noservice`, hence this wishlist report.
